src/unix.rs

```diff
@@ -1,63 +1,43 @@
 extern crate libc;
 
-use std;
 use std::io;
-use std::process::Child;
 
-// A handle on Unix is just the PID.
-pub struct Handle(u32);
-
-pub fn get_handle(child: &Child) -> Handle {
-    Handle(child.id())
+pub trait SharedChildExt {
+    /// Sends a signal to the child process with `libc::kill`.
+    fn send_signal(&self, signal: libc::c_int) -> io::Result<()>;
```

Comment:
I was about to suggest just calling this `signal`, but I guess that's the name of the libc function for _handling_ signals, so actually I like it this way :)